parts/linux/cloud-init/nodecustomdata.yml

```
@@ -300,6 +300,16 @@ write_files:
     #EOF
 {{- end}}
 
+{{- if ShouldConfigureCustomCATrust}}
+{{range $i, $cert := GetCustomCATrustConfigCerts}}
+- path: /usr/local/share/ca-certificates/00000000000000cert{{$i}}.crt
+  permissions: "0644"
+  owner: root
+  content: |
+  {{$cert}}
```

```suggestion
  {{$cert}}
  #EOF
```

annoyingly this is how wait_for_file checks if cloud init finished writing it to disk

---

also something seems a tiny bit of with the indentation but I can't tell what or if it will even matter - https://github.com/Azure/AgentBaker/blob/8166500194be7a250b2943a58c03a447309c4e98/pkg/agent/testdata/AKSUbuntu1804%2BCustomCATrust/CustomData#L92-L113

look at the previous file 99periodic vs yours